examples/resources/genesyscloud_script/apis.md

```diff
@@ -0,0 +1,5 @@
+* [POST /api/v2/scripts/{scriptId}/export](https://developer.genesys.cloud/devapps/api-explorer#post-api-v2-scripts--scriptId--export)
+* [GET /api/v2/scripts](https://developer.genesys.cloud/devapps/api-explorer#get-api-v2-scripts)
+* [GET /api/v2/scripts/{scriptId}](https://developer.genesys.cloud/devapps/api-explorer#get-api-v2-scripts--scriptId-)
+* [GET /api/v2/scripts/uploads/{uploadId}/status](https://developer.genesys.cloud/devapps/api-explorer#get-api-v2-scripts-uploads--uploadId--status)
+* DELETE /api/v2/scripts/{scriptId}
```

Comment:
Would it be better to omit this line? Since this endpoint isn't publicly listed yet

Reply:
@dginty4 @carnellj

Reply:
Omit the line if it is not public